### PR TITLE
Add section for community chat rooms

### PR DESCRIPTION
**Pull request by jamesmunns**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,13 +68,21 @@ This project is developed and maintained by the [Resources team][team].
...
 
 - [Embedded WG](https://github.com/rust-embedded/wg/), including newsletters with progress updates.
 
-- You can usually find community members (including embedded WG members) in the [`#rust-embedded:matrix.org` Matrix room](https://matrix.to/#/#rust-embedded:matrix.org).
+### Community Chat Rooms
 
-- [embedded.rs](https://t.me/embedded_rs) Telegram chat about Rust for microcontrollers in Russian language.
+- You can usually find community members (including embedded WG members) in the official [`#rust-embedded:matrix.org` Matrix room].
+- [embedded.rs] - Telegram chat about Rust for microcontrollers in the Russian language.
+- [#rust-embedded-graphics:matrix.org] - For discussion of the [`embedded-graphics`] crate and ecosystem
```
Comment (jamwaffles) [inline README.md]: This needs a link to maybe crates.io?

```suggestion
- [#rust-embedded-graphics:matrix.org] - For discussion of the [`embedded-graphics`](https://crates.io/crates/embedded-graphics) crate and ecosystem
```

Comment (jamesmunns) [reply README.md]: Is the link bad? I thought there was a link lower down in the page already (in the `no_std` section)

**Pull request merged**